index.bs

```diff
@@ -4776,6 +4782,49 @@ RogueList when performing ECDAA-Verify (see section 3.6 in [[!FIDOEcdaaAlgorithm
 [[FIDOMetadataService]] provides one way to access such information.
 
 
+## Security Benefits for [RPS] ## {#sctn-rp-benefits}
+
+The main benefits offered to [=[RPS]=] by this specification include:
+
+1. Users and accounts can be secured using widely compatible, easy-to-use multi-factor authentication.
+1. The [=[RP]=] does not need to provision [=authenticator=] hardware to its users. Instead, each user can independently obtain
+    any conforming [=authenticator=] and use that same [=authenticator=] with any number of [=[RPS]=]. The [=[RP]=] can optionally
+    enforce requirements on [=authenticators=]' security properties by inspecting the [=attestation statements=] returned from the
+    [=authenticators=].
+1. [=Registration=] and [=authentication=] [=ceremonies=] are highly resistant to [=man-in-the-middle attacks=].
+1. The [=[RP]=] can automatically support multiple types of [=user verification=] - for example PIN, biometrics and/or future
+    methods - with little or no code change, and can let each user decide which they prefer to use via their choice of
+    [=authenticator=].
```

Comment:
yeah, we can make the above statement but it leaves out the hard work a deployer has to (or ought to, anyway) do in order to provide for account recovery modulo users' use of a multiplicity of authenticators (some 1st-multi-factor, some 2nd-factor-only) and also train customer support personnel wrt the multiplicity of user verification modalities.  Maybe the latter will end up not being that bad, but I have heard some deployers thinking that for these reasons they would prefer to shepherd all their users to use (primarily) one user verif modality (eg fingerprint).  Again, maybe as the authnr matchers improve and become ubiquitous, and users get used to using them to unlock their devices in the first place, this wont be as much of a concern.... but I wonder if we ought to mention some of these deployment facets in the spec (deploy-cons).

Reply:
Good points. Would you prefer we just strike this point off the list?